src/components/Reports/ExpectedMonthlyTotalReport/Table/ExpectedMonthlyTotalReportTable.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useState } from 'react';
+import {
+  Box,
+  Typography,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Paper,
+  Accordion,
+  AccordionSummary,
+  AccordionDetails,
+  Link,
+} from '@material-ui/core';
+import { ExpandMore } from '@material-ui/icons';
+import { useTranslation } from 'react-i18next';
+
+export interface Contact {
+  contact: string;
+  contactId: string;
+  status: string;
+  commitment: string;
+  frequency: string;
+  converted: string;
+  currency: string;
+  donation?: string;
+}
+
+interface Props {
+  title: string;
+  data: Contact[];
+  donations: boolean;
+}
+
+export const ExpectedMonthlyTotalReportTable: React.FC<Props> = ({
+  title,
+  data,
+  donations,
+}) => {
+  const { t } = useTranslation();
+
+  const totalPartners = () => {
+    return 6;
+  };
+
+  const [visible, setVisible] = useState(true);
+
+  const total = () => {
+    return 3920;
+  };
+
+  const showTotalPartners = () => {
+    if (visible) {
+      return t(' Show ' + totalPartners() + ' Partners');
+    }
+  };
+  return (
+    <Box
+      style={{
+        maxWidth: '98%',
+        padding: 4,
+        margin: 'auto',
+      }}
+    >
+      <Accordion
+        style={{ marginLeft: 0 }}
+        onClick={() => setVisible((v) => !v)}
+      >
+        <AccordionSummary
+          expandIcon={<ExpandMore />}
+          style={{
+            backgroundColor: '#f0f0f0',
```

Comment:
Above, add this: 
`import theme from '../../../../theme.ts'`

Then change this line to:

```suggestion
            backgroundColor: theme.palette.cruGrayLight.main,
```